src/find-files-with-gql.js

```diff
@@ -17,10 +17,13 @@ export const findRepoRoot = (): string => {
     }
 };
 
-export const findGraphqlTagReferences = (root: string): Array<string> => {
+export const findGraphqlTagReferences = (
+    root: string,
+    needle: string = 'gql`',
```

Comment:
I doubt the name `needle` will be helpful to people who haven't seen the `needle/haystack` metaphor in code before. Maybe `search` or `tag` would be a better name? The function signature is also straddling two levels of abstraction now: `findGraphqlTagReferences` claims to be about `gql` tags, but it actually just finds JavaScript files containing the given string.

Since it seems no one is using the `needle` parameter yet, I think it could be added in a separate change.